core/corerepo/gc.go

```diff
@@ -15,54 +16,40 @@ type KeyRemoved struct {
 }
 
 func GarbageCollect(n *core.IpfsNode, ctx context.Context) error {
```

Comment:
we've always had the `core.*` functions take the node as their first argument. You want to change this everywhere?